### ISSUE TITLE
SERVICE fails in empty context.

### ISSUE THREAD

**Issue by afs** [apache/jena#1399]
### Version

4.6.0-SNAPSHOT

### What happened?

Executing a query involving `SERVICE` with an empty context does not work because there is no default service execution.

@Aklakan - for your improvements in this area, could you please ensure that it executes when the context does not have a setting for `ARQConstants.registryServiceExecutors`.

### Relevant output and stacktrace

```shell
public static void main(String...args) {
        FusekiLogging.setLogging();
        FusekiServer server = FusekiServer.create()
                .port(0)
                .add("/ds",  DatasetGraphFactory.empty())
                .start();
        String SERVICE = "http://localhost:"+server.getPort()+"/ds";
        Node expected = NodeFactory.createLiteral("28181", XSDDatatype.XSDinteger);
        String queryString = "SELECT * { SERVICE <"+SERVICE+"> { VALUES ?x { 28181 } } }";
        // Empty context
        Context cxt = new Context();
        //Context cxt = ARQ.getContext().copy();
        RowSet rs = QueryExec.dataset(DatasetGraphFactory.empty())
                .query(queryString)
                .context(cxt)
                .select()
                .materialize();
        assertTrue(rs.hasNext());
        Node n = rs.next().get("x");
        assertEquals(expected, n);
    }
```

**Comment by Aklakan** [comment]
So the rules should be like follows - right?

- empty context &rarr; default behavior i.e. http service request should work
- context with present but empty service executor registry &rarr; error because no registered executor

**Comment by Aklakan** [comment]
I added a fix resolution within #1388 - will test tomorrow.

**Comment by afs** [comment]
If you add "This fixes #1399" to the description, magic happens and this issue is automatically closed.
